2-Authorization-I/1-call-graph/README.md

```diff
@@ -31,15 +31,19 @@ Here you'll learn how to [sign-in](https://docs.microsoft.com/azure/active-direc
 
 ## Contents
 
-| File/folder         | Description                                                                |
-|---------------------|----------------------------------------------------------------------------|
-| `App.jsx`           | Main application logic resides here.                                       |
-| `fetch.jsx`         | Provides a helper method for making fetch calls using bearer token scheme. |
-| `graph.jsx`         | Instantiates Graph SDK client using a custom authentication provider.      |
-| `authConfig.js`     | Contains authentication configuration parameters.                          |
-| `pages/Profile.jsx` | Calls Microsoft Graph `/me` endpoint vith Graph SDK.                       |
-| `pages/Mails.jsx`   | Calls Microsoft Graph `/me/messages` endpoint vith Graph SDK.              |
-| `pages/Tenants.jsx` | Calls Microsoft Graph `/tenants` endpoint via fetch API.                   |
+| File/folder                         | Description                                                                |
+|-------------------------------------|----------------------------------------------------------------------------|
+| `App.jsx`                           | Main application logic resides here.                                       |
+| `fetch.jsx`                         | Provides a helper method for making fetch calls using bearer token scheme. |
+| `graph.jsx`                         | Instantiates Graph SDK client                                              |
+| `authConfig.js`                     | Contains authentication configuration parameters.                          |
+| `pages/Home.jsx`                    | Contains a table with ID token claims and description                      |
+| `pages/Redirect.jsx`                | Blank page for redirect purposes. When using popup and silent APIs         |
+| `pages/Profile.jsx`                 | Calls Microsoft Graph `/me` endpoint vith Graph SDK.                       |
+| `pages/Mails.jsx`                   | Calls Microsoft Graph `/me/messages` endpoint vith Graph SDK.              |
```

Comment:
>Mails [](http://example.com/codeflow?start=9&length=5)

replace with "Contacts" as mails is a very troublesome API

Reply:
Done